docs/structured-streaming-programming-guide.md

```diff
@@ -14,29 +14,13 @@ Structured Streaming is a scalable and fault-tolerant stream processing engine b
 
 # Quick Example
 Let’s say you want to maintain a running word count of text data received from a data server listening on a TCP socket. Let’s see how you can express this using Structured Streaming. You can see the full code in 
-[Scala]({{site.SPARK_GITHUB_URL}}/blob/master/examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredNetworkWordCount.scala)/
-[Java]({{site.SPARK_GITHUB_URL}}/blob/master/examples/src/main/java/org/apache/spark/examples/sql/streaming/JavaStructuredNetworkWordCount.java)/
-[Python]({{site.SPARK_GITHUB_URL}}/blob/master/examples/src/main/python/sql/streaming/structured_network_wordcount.py). And if you 
-[download Spark](http://spark.apache.org/downloads.html), you can directly run the example. In any case, let’s walk through the example step-by-step and understand how it works. First, we have to import the necessary classes and create a local SparkSession, the starting point of all functionalities related to Spark.
+[Scala]({{site.SPARK_GITHUB_URL}}/blob/master/examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredNetworkWordCount.scala)/[Java]({{site.SPARK_GITHUB_URL}}/blob/master/examples/src/main/java/org/apache/spark/examples/sql/streaming/JavaStructuredNetworkWordCount.java)/[Python]({{site.SPARK_GITHUB_URL}}/blob/master/examples/src/main/python/sql/streaming/structured_network_wordcount.py). And if you 
+[download Spark](http://spark.apache.org/downloads.html), you can directly run the example. In any case, let’s walk through the example step-by-step and understand how it works.
 
 <div class="codetabs">
 <div data-lang="scala"  markdown="1">
 
-
```

Comment:
Before | After
----------|---------
![image](https://cloud.githubusercontent.com/assets/8685962/16900343/dc61ee0e-4c22-11e6-8ae3-58b7f6d70ca6.png) | ![image](https://cloud.githubusercontent.com/assets/8685962/16900349/014fd1b8-4c23-11e6-882f-4421cbfbb5fe.png)

# Before
- "First, we have to ..." should be moved to the corresponding code block
- The "Next, let's create ..." paragraph is repeated twice

![image](https://cloud.githubusercontent.com/assets/8685962/16900343/dc61ee0e-4c22-11e6-8ae3-58b7f6d70ca6.png)


# After
![image](https://cloud.githubusercontent.com/assets/8685962/16900349/014fd1b8-4c23-11e6-882f-4421cbfbb5fe.png)